e2e/test/iothub/messaging/MessageSendFaultInjectionTests.cs

```diff
@@ -95,161 +90,131 @@ public async Task Message_TcpConnectionLossSendRecovery_MqttWs()
+                FaultInjection.DefaultFaultDelay).ConfigureAwait(false);
         }
 
         [LoggedTestMethod, Timeout(TestTimeoutMilliseconds)]
         public async Task Message_AmqpD2CLinkDropSendRecovery_AmqpWs()
         {
             await SendMessageRecoveryAsync(
-                    TestDeviceType.Sasl,
-                    new IotHubClientAmqpSettings(IotHubClientTransportProtocol.WebSocket),
-                    FaultInjection.FaultType_AmqpD2C,
-                    FaultInjection.FaultCloseReason_Boom,
-                    FaultInjection.DefaultFaultDelay)
-                .ConfigureAwait(false);
+                TestDeviceType.Sasl,
+                new IotHubClientAmqpSettings(IotHubClientTransportProtocol.WebSocket),
+                FaultInjection.FaultType_AmqpD2C,
+                FaultInjection.FaultCloseReason_Boom,
+                FaultInjection.DefaultFaultDelay).ConfigureAwait(false);
         }
 
         [LoggedTestMethod, Timeout(TestTimeoutMilliseconds)]
         public async Task Message_ThrottledConnectionRecovery_Amqp()
         {
             await SendMessageRecoveryAsync(
-                    TestDeviceType.Sasl,
-                    new IotHubClientAmqpSettings(),
-                    FaultInjection.FaultType_Throttle,
-                    FaultInjection.FaultCloseReason_Boom,
-                    FaultInjection.DefaultFaultDelay,
-                    FaultInjection.DefaultFaultDuration)
-                .ConfigureAwait(false);
+                TestDeviceType.Sasl,
+                new IotHubClientAmqpSettings(),
+                FaultInjection.FaultType_Throttle,
+                FaultInjection.FaultCloseReason_Boom,
+                FaultInjection.DefaultFaultDelay,
+                FaultInjection.DefaultFaultDuration).ConfigureAwait(false);
         }
 
         [LoggedTestMethod, Timeout(TestTimeoutMilliseconds)]
         [TestCategory("Flaky")]
         public async Task Message_ThrottledConnectionRecovery_AmqpWs()
         {
             await SendMessageRecoveryAsync(
-                    TestDeviceType.Sasl,
-                    new IotHubClientAmqpSettings(IotHubClientTransportProtocol.WebSocket),
-                    FaultInjection.FaultType_Throttle,
-                    FaultInjection.FaultCloseReason_Boom,
-                    FaultInjection.DefaultFaultDelay,
-                    FaultInjection.DefaultFaultDuration)
-                .ConfigureAwait(false);
+                TestDeviceType.Sasl,
+                new IotHubClientAmqpSettings(IotHubClientTransportProtocol.WebSocket),
+                FaultInjection.FaultType_Throttle,
+                FaultInjection.FaultCloseReason_Boom,
+                FaultInjection.DefaultFaultDelay,
+                FaultInjection.DefaultFaultDuration).ConfigureAwait(false);
         }
 
         [LoggedTestMethod, Timeout(TestTimeoutMilliseconds)]
+        [ExpectedException(typeof(IotHubThrottledException))]
```

Comment:
Update the test to expect only IotHubThrottledException and remove generic timeout related exceptions as acceptable expected exceptions.